lib/functions.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+die() {
+    # Print in RED
+    printf "\e[31m==> [ERROR] %s \e[0m\n" "${1:-'unknown'}"
+    exit 1
+}
+
+success() {
+    # Print in GREEN
+    printf "\e[32m==> [SUCCESS] %s\e[0m\n" "${1:-'unknown'}"
+    exit 0
+}
+
+warn() {
+    # Print in ORANGE
+    printf "\e[33m==> [WARN] %s\e[0m\n" "${1:-'unknown'}"
+}
+
+info() {
+    # Print in BLUE
+    printf "\e[96m==> [INFO] %s\e[0m\n" "${1:-'unknown'}"
+}
+
+ocp_sanity_check() {
+    if [ ! -x $(command -v oc) ]; then
+        die "Could not find the executable oc in the current PATH"
+    fi
+
+    whoami=$(oc whoami 2> /dev/null)
```

Comment:
does oc whoami work when using a kubeconfig env variable?
we should edit the message to indicate to set KUBECONFIG env variable when this fails

Reply:
Will do, thanks.